components/financial_assistance/app/views/financial_assistance/applicants/tax_info.html.erb

```diff
@@ -207,7 +207,7 @@
           <div class="col-xs-12">
             <div class="back-link">
               <i class="fa fa-caret-left fa-2x" aria-hidden="true"></i>
-              <a href="javascript:void(0);" onclick="notifyUserPrompt(this)" data-path="<%= financial_assistance.edit_application_path(@application) %>"> <b><%= l10n("faa.tax_info.back") %></b> </a> <span><%= l10n("faa.tax_info.back_paranthetical") %></span>
+              <a href="javascript:void(0);" class="back-to-household-link" data-path="<%= financial_assistance.edit_application_path(@application) %>"> <b><%= l10n("faa.tax_info.back") %></b> </a> <span><%= l10n("faa.tax_info.back_paranthetical") %></span>
```

Comment:
replace `javascript:void(0)` with `#`. Should add       `event.preventDefault()` to the top of the function call if it isn't there already. I don't think the javascript:void will pass the CSP test.